seed/challenges/01-front-end-development-certification/basic-javascript.json

```diff
@@ -2419,6 +2427,10 @@
         "",
         "// Only change code below this line",
         "",
+        "",
+        "// Only change code above this line",
+        "",
+        "timesFive(5);",
```

Comment:
All they need to type in is the function definition, all of the challenges pass without the function call on the bottom at all. Although they do not pass if that function is not defined at all. I tend to look on the bottom function call in the lessons as just a helper (in fact, I usually would wrap it in a console log while I was working on the lesson and never had any issues). In the case of these where the camper is asked to create a function of that name, the fact that it won't pass without the function defined properly is an added little data point for the camper.